src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -15,48 +15,51 @@
   [:div {:style {:padding "1em 0 2em 0"}} children])
 
 (defn view-attributes [state refs]
-  [:div {:style {:display "inline-block"}}
+  [:div {}
    (style/create-section-header "Workspace Attributes")
    (create-section
      (when (or (:saving? @state) (:deleting? @state))
        [comps/Blocker {:banner "Updating..."}])
      [:div {}
       (map-indexed
-        (fn [i a]
-          [:div {}
-           [:div {:style {:float "left" :marginRight "0.5em"}}
+        (fn [i [attr-key attr-value]]
+          [:div {:style {:display "flex" :alignItems "baseline"}}
+           [:div {:style {:flex "1 0 auto" :marginRight "0.5em"}}
```

Comment:
maybe go for something like `:flex-basis "35%"` here, instead of making both columns equal? I like the end result of this PR much better than previous state, but after this PR the fields can almost be TOO big. I argue that the attr key needs less space than the attr value, so something like 35%:65% might be good. Thoughts?